views/templates/hook/blockreassurance.tpl

```diff
@@ -20,7 +20,12 @@
     <div id="block-reassurance">
         <ul>
             {foreach from=$elements item=element}
-                <li><img src="{$element.image}" alt="{$element.text|escape:'quotes'}"/> <span>{$element.text}</span></li>
+                <li>
+                    <div class="block-reassurance-item" {if $element['type_link'] !== $LINK_TYPE_NONE && !empty($element['link'])} style="cursor:pointer;" onclick="window.open('{$element['link']}')"{/if}>
```

Comment:
* You have a CSS class, so you can put the style in this class. 
* I'm not fan of the `onclick` method to emulate a link

Reply:
I don't get the reason why people who developed this didn't use links in the past, is there a valid reason?

Reply:
@NeOMakinG Neither do I. By the way, we have 3 template files with `onclick`, so one more won't make this module worse.
Solving the `onclick` in another issue and PR will be safer and make less confuses.